Monika After Story/game/script-topics.rpy

```diff
@@ -11780,7 +11781,363 @@ label monika_stoicism:
     return
 
 init 5 python:
+
     addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_fun_facts_open",
```

Comment:
we just moved fun facts to a different file, you need to rebranch or update your branch with the latest changes.